joomla/joomla-cms
# [4.3] Batch Copy Banners

Pull Request for Issue #40261 .

### Summary of Changes
ensure that $published is an integer


### Testing Instructions
Create a banner and then try a batch action. It should look like this
![image](https://user-images.githubusercontent.com/1296369/229096578-0a6c868d-72b3-400f-aa9b-c9110aaef9f1.png)

Then apply any filter and/or sort by any column and try to batch action again

### Actual result BEFORE applying this Pull Request

![image](https://user-images.githubusercontent.com/1296369/229096798-5409ebc9-b928-4992-b08c-73ccb7e397ec.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/229096810-cd2caa1c-23a0-4f26-ab76-0d020570e57c.png)



### Link to documentations
Please select:
